pints/tests/test_toy_stochastic_logistic_model.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+#
+# Tests if the stochastic logistic growth (toy) model works.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import unittest
+import numpy as np
+import pints
+import pints.toy
+from pints.toy import StochasticLogisticModel
+
+
+class TestStochasticLogistic(unittest.TestCase):
+    """
+    Tests if the stochastic logistic growth (toy) model works.
+    """
+    def test_start_with_zero(self):
```

Comment:
If this is a stochastic method, we should seed the random generator and compare output values (search other tests for `seed(` to find examples)

Reply:
yep agreed